src/main/resources/db/migration/V4__user_files_change_primary_key_type.sql

```diff
@@ -0,0 +1,7 @@
+-- This will make the original file_id a column a uuid column
+create extension if not exists "uuid-ossp";
```

Comment:
In theory postgreSQL vs 14 (which I think we use) should have this function that we could use w/o creating an extension: 
```
alter table user_files add column new_id uuid DEFAULT gen_random_uuid();
```

I've tried (and failed) to get that variant to work with tests.

Reply:
From what I can see flyway is trying to instantiate a bean of the db stuff, and that's causing migrations to run, and fail.  It's not really the tests themselves, I think.